nikola/plugins/command/import_wordpress.py

```diff
@@ -196,9 +196,30 @@ class CommandImportWordpress(Command, ImportMixin):
             'default': 'first',
             'help': 'lower: Convert all tag and category names to lower case\nfirst: Keep first spelling of tag or category name',
         },
+        {
+            'name': 'one_file',
+            'long': 'one-file',
+            'default': False,
```

Comment:
Well, it would be a problem if someone re-imports the same wordpress dump
over his site, but it's just a corner case...

On Tue, Mar 8, 2016 at 1:36 PM Chris Warrick <notifications@github.com>
wrote:

> In nikola/plugins/command/import_wordpress.py
> <https://github.com/getnikola/nikola/pull/2292#discussion_r55385856>:
>
> > @@ -196,9 +196,30 @@ class CommandImportWordpress(Command, ImportMixin):
> >              'default': 'first',
> >              'help': 'lower: Convert all tag and category names to lower case\nfirst: Keep first spelling of tag or category name',
> >          },
> > +        {
> > +            'name': 'one_file',
> > +            'long': 'one-file',
> > +            'default': False,
>
> We could probably default to True.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getnikola/nikola/pull/2292/files#r55385856>.
>